lib/capybara/node/element.rb

```diff
@@ -429,17 +430,17 @@ def drop(*args)
 
       ##
       #
-      # Scroll the page or element
+      # Scroll the page or element.
       #
-      # Scroll the page or element to its top, bottom or middle
       # @overload scroll_to(position, offset: [0,0])
+      #   Scroll the page or element to its top, bottom or middle.
       #   @param [:top, :bottom, :center, :current] position
-      #   @param :offset
+      #   @param [[Integer, Integer]] offset
```

Comment:
[note] I add a type to the `offset` parameter, but I'm worried a bit whether it's correct… 🤔

Reply:
The type is correct - please note that it defaults to [0,0]

Reply:
Thanks 👍